### ISSUE TITLE
Feature request: Instead of always writing to file, allow reading/writing to in memory object for streaming in Flask

### ISSUE THREAD

**Issue by Code4SAFrankie** [ArtifexSoftware/pdf2docx#223]
Instead of always writing to file, allow reading of the PDF from a memory object and then writing of the DOCX output to a memory object. This is in order to use the result as a streaming object in Flask/FastAPI etc. I would like to avoid writing to a temporary file, since I am not sure how that works in Azure App Service Basic 1 Plan or Docker which might well be immutable. Also there are limits on basic plans so you would need to implement additional cleanup code.

Thanks!

**Comment by dothinking** [comment]
Hi Code4SAFrankie, thanks for your good suggestion.
This feature was already implemented in the master branch but didn't publish to pypi yet. Will publish `v0.5.7` soon.
Before that, please refer to #108 and #177.

**Comment by dothinking** [comment]
with code in the latest master branch, you can use stream object like:

```python
import io
from pdf2docx import Converter

with open('sample.pdf', 'rb') as f: in_stream = f.read()
out_stream = io.BytesIO()

#-------------------------------------------

cv = Converter(stream=in_stream)
cv.convert(out_stream)

#-------------------------------------------

with open('sample.docx', 'wb') as f: f.write(out_stream.getvalue())
```

**Comment by dothinking** [comment]
Implemented in `v0.5.7`.
